iceoryx_hoofs/test/moduletests/test_cxx_variant.cpp

```diff
@@ -703,4 +703,41 @@ TEST_F(variant_Test, CopyVariantIntoVariantOfDifferentType)
 
     EXPECT_THAT(DoubleDelete::dtorCalls, Eq(1));
 }
+
+TEST_F(variant_Test, TwoInvalidVariantsAreEqual)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "4b2b7516-48ef-4aa6-a0c5-43a204e1e348");
+    iox::cxx::variant<std::string, float> sut1;
+    iox::cxx::variant<std::string, float> sut2;
+    EXPECT_TRUE(sut1 == sut2);
```

Comment:
Please add an EXPECT_*(...) for the inequality comparison in every test like
EXPECT_FALSE(sut1 != sut2);